.github/workflows/integration-test.yml

```diff
@@ -39,6 +39,8 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+      with:
+        submodules: 'recursive'
```

Comment:
This makes the integration test run longer because it gets all submodules.
The code is written to only checkout out the submodules it verifies

Reply:
I was in the process of moving away from submodules towards just using `git clone` during the execution.